frontend/lib/src/components/shared/StreamlitMarkdown/Heading.tsx

```diff
@@ -62,64 +62,39 @@ function Heading(props: HeadingProtoProps): ReactElement {
   const [heading, ...rest] = body.split("\n")
 
   return (
-    <div className="stHeadingContainer" data-testid="stHeading">
-      <div className="stMarkdown" style={{ width }}>
-        <StyledStreamlitMarkdown
-          isCaption={Boolean(false)}
-          isInSidebar={isInSidebar}
-          style={{ width }}
-          data-testid="stMarkdownContainer"
+    <div className="stMarkdown" style={{ width }} data-testid="stHeading">
```

Comment:
hmm, maybe its better to use `stHeading` also for className. `stMarkdown` is a bit misleading since its not orginated by `st.markdown`

Reply:
I have removed the classname here completely for now 👍